dist/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Meals app</title>
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+<script defer src="index.bundle.js"></script></head>
+
+<body>
+  <nav class="container border border-dark border-3 mt-4 navbar navbar-expand navbar-light bg-white menu">
+    <div class="container-fluid col-12 col-md-12">
+      <a href="#home"><img src="#" alt="" id="logo" class="navbar-brand col-4"></a>
+      <ul class="navbar-nav ms-auto d-flex align-items-baseline col-8 justify-content-end">
+        <li class="nav-item mx-lg-2 mx-xl-3 mx-xxl-4">
+          <a class="nav-link" href="#home" id="list-identifier">Home</a>
+        </li>
+        <li class="nav-item mx-lg-2 mx-xl-3 mx-xxl-4">
+          <a class="nav-link" href="#ingredients" id="add-identifier">Ingredients</a>
+        </li>
+        <li class="nav-item ml-lg-2 ml-xl-3 ml-xxl-4">
+          <a class="nav-link" href="#about" id="contact-identifier">About</a>
+        </li>
+      </ul>
+    </div>
+  </nav>
+  <section class="container-fluid my-4 py-3">
+    <div class="row gx-5 justify-content-center">
+      <div class="col-3 mx-3 card d-flex align-content-around">
+        <img src="https://www.themealdb.com/images/media/meals/xvsurr1511719182.jpg" alt="Apple Crumble">
+        <h2 class="py-2">Apple & Blackberry Crumble</h2>
+        <input type="submit" name="comments" id="comments-0" value="Comments"
+          class="btn comment text-light btn-block my-2 ">
+        <input type="submit" name="reservations" id="reservation-0" value="Reservations"
+          class="btn reserve  text-light my-2 btn-block">
+
+      </div>
+      <div class="col-3 mx-3 card d-flex align-content-between">
+        <img src="https://www.themealdb.com/images/media/meals/adxcbq1619787919.jpg" alt="">
+        <h2 class="py-2">Apam balik</h2>
+        <input type="submit" name="comments" id="comments-1" value="Comments"
+          class="btn comment text-light btn-block my-2">
+        <input type="submit" name="reservations" id="reservation-1" value="Reservations"
+          class="btn reserve text-light my-2 btn-block">
+
+      </div>
+      <div class="col-3 mx-3 card d-flex align-content-around">
+        <img src="https://www.themealdb.com/images/media/meals/wxywrq1468235067.jpg" alt="">
+        <h2 class="py-2">Apple Frangipan Tart</h2>
+        <input type="submit" name="comments" id="comments-2" value="Comments"
+          class="btn comment text-light btn-block my-2">
+        <input type="submit" name="reservations" id="reservation-2" value="Reservations"
+          class="btn reserve text-light my-2 btn-block">
```

Comment:
Add likes button to meet the requirement according to the design

![image](https://user-images.githubusercontent.com/68148157/143072466-8ab1791a-2ab7-49b8-a542-faba442742a8.png)

Reply:
Done!